client/lib/checkout/payment-methods.ts

```diff
@@ -28,6 +28,8 @@ export interface PaymentMethod {
 	remember: '1' | '0';
 	stored_details_id: string;
 	user_id: string;
+	tax_postal_code: string;
```

Comment:
I think these changes were left-over from a previous iteration and can be removed. AFAIK we aren't using this type anywhere.

Reply:
Removed.